misaligned/MisAlignedColorWirePair.java

```diff
@@ -0,0 +1,32 @@
+import java.util.List;
+
+public class MisAlignedColorWirePair {
+
+     public static final String WHITE="White";
+     public static final String BROWN="Brown";
+
+    public static void main(String[] args) {
+        List<ColorWirePair> listColorWirePair = getColorWirePair();
+        colorWirePairTest(listColorWirePair);
+        ColorWirePairPrinter colorWirePairPrinter = new ColorWirePairPrinter();
+        colorWirePairPrinter.printColorWirePair(listColorWirePair);
+        int colorWirePairSet = listColorWirePair.size();
+        assert (colorWirePairSet !=25) : "Valid color wire pairs";
+        System.out.println("All is well (maybe!)");
+
+    }
+
+    private static void colorWirePairTest(final List<ColorWirePair> listColorWirePair){
+        ColorWirePair colorWirePair= listColorWirePair.get(3);
```

Comment:
Boundary values also need to be considered while writing test cases. for ex. pair number 1, 5, 6, 10,..25